process-controller/src/main/java/org/jboss/as/process/ManagedProcess.java

```diff
@@ -107,7 +125,9 @@ enum State {
         this.pcAuthKey = pcAuthKey;
         isPrivileged = privileged;
         respawnPolicy = respawn ? RespawnPolicy.RESPAWN : RespawnPolicy.NONE;
-        log = Logger.getMessageLogger(ProcessLogger.class, "org.jboss.as.process." + processName + ".status");
+        logStatus = Logger.getMessageLogger(ProcessLogger.class, "org.jboss.as.process." + processName + ".status");
+        logSystemErr = Logger.getMessageLogger(ProcessLogger.class, "org.jboss.as.process." + processName + ".system.sterr");
+        logSystemOut = Logger.getMessageLogger(ProcessLogger.class, "org.jboss.as.process." + processName + ".system.stout");
```

Comment:
s/stout/stdout

Reply:
+1 to 

"org.jboss.as.process." + processName + "<suffix>"

We don't want to mess with the existing 'status' and these should be consistent.

Reply:
Done, thanks!